samples/integration/vcpkg-keyvault/.devcontainer/devcontainer.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "Ubuntu-21.04",
+    "dockerFile": "Dockerfile",
+    "settings": { 
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "extensions": [
+		"ms-vscode.cpptools-themes",
+		"bbenoist.doxygen",
+		"streetsidesoftware.code-spell-checker",
+		"ms-vscode.cpptools",
+		"xaver.clang-format",
+		"twxs.cmake",
+		"ms-vscode.cmake-tools",
+		"eamodio.gitlens",
+		"davidschuldenfrei.gtest-adapter"
+	],
+    // Do not run as root. See https://aka.ms/vscode-remote/containers/non-root.
+    "remoteUser": "azure-sdk-for-cpp"
```

Simplier and easier name to remember
```suggestion
    "remoteUser": "azure-user"
```

---

There is  relation between the image that is pulled from docker hub (vhvb1989/keyvault:beta1) and this value.

The image has created a linux user with this name already.  Changing the name would break vscode workspace to work, until the image from the docker hub is updated.   

And the user name is not really such relevant!  What we want is a ready-to-use dev env where to run the samples